english/plugh/outside/outside1.md

```diff
@@ -4,7 +4,7 @@ Password is still in your pocket. Weather is nice, birds are tweeting.
 
 *Pstst*. Can of soda is opened. You drink slowly and enjoy it. 
 
-However your inner voice stars raising up. You think you noticed some movement. 
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.